backend/models/travel.ts

```
@@ -180,29 +182,26 @@ travelSchema.methods.saveToHistory = async function (this: TravelDoc) {
-                }
-              }
+  this.log[this.state] = { date: new Date(), editor: this.editor }
+
+  if (this.state === 'approved') {
+    // move vehicle registration of owner as receipt to 'ownCar' stages
+    const receipts = []
+    for (const stage of this.stages) {
+      if (stage.transport.type == 'ownCar') {
+        if (receipts.length == 0) {
+          const owner = await User.findOne({ _id: this.owner._id }).lean()
+          if (owner && owner.vehicleRegistration) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)